dbflow/src/main/java/com/raizlabs/android/dbflow/structure/ModelAdapter.java

```diff
@@ -63,12 +57,8 @@ public DatabaseStatement getInsertStatement(DatabaseWrapper databaseWrapper) {
      * @return The precompiled full statement for this table model adapter
      */
     public DatabaseStatement getCompiledStatement() {
-        if (compiledStatement == null) {
-            compiledStatement = getCompiledStatement(
-                    FlowManager.getDatabaseForTable(getModelClass()).getWritableDatabase());
-        }
-
-        return compiledStatement;
+        return getCompiledStatement(
```

Comment:
#req not sure why this is no longer cached, but we should also cache it. This is not really used anywhere, more for convenience.